pcip/README.md

```diff
@@ -0,0 +1,90 @@
+# Pulsar Java Contrib Improvement Proposal (PCIP)
+
+## What is a PCIP?
+
+The PCIP is a "Pulsar Java Contrib Improvement Proposal" and it's the mechanism used to propose changes to the Apache Pulsar Java Contrib codebases.
+
+The changes might be in terms of new features, large code refactoring, changes to APIs.
+
+In practical terms, the PCIP defines a process in which developers can submit a design doc, receive feedback and get the "go ahead" to execute.
+
+
+### What is the goal of a PCIP?
+
+There are several goals for the PCIP process:
```

Comment:
IMO, the goals can be divided into:
1. As a user manual, add instructions when introducing new features or modifying existing features.
2. Explain the functional design ideas to facilitate review and later maintenance.

Reply:
done